packages/electrode-archetype-react-app/lib/features.js

```diff
@@ -9,6 +9,7 @@ const os = require("os");
 const exec = util.promisify(require("child_process").exec);
 const appArchetypeConfig = optionalRequire("electrode-archetype-react-app/config/archetype") || {};
 const prompts = require("prompts");
+const request = require("request");
```

Comment:
`request` is dep in `electrode-archetype-react-app-dev`.  we use `appArchetypeConfig.devRequire` to ensure start looking from there.